vendor/gwf/rcm-lrw10-868.yaml

```diff
@@ -0,0 +1,7 @@
+vendorProfileID: 01
+supportsClassB: false
+supportsClassC: false
+macVersion: 1.0.1
```

Comment:
Hi @maebli you should add `regionalParametersVersion`

Reply:
@Jaime-Trinidad thanks for the review!

I thought that for macVersion 1.0.1 the Regional Parameters are not a separate document. 
I saw the following in another vendor folder:

```
# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
#   1.0:   TS001-1.0
#   1.0.1: TS001-1.0.1
#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
#   1.0.3: RP001-1.0.3-RevA
#   1.0.4: RP002-1.0.0 or RP002-1.0.1
#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
regionalParametersVersion: RP001-1.0.2
```

and I tried to build using
regionalParametersVersion: TS001-1.0

but the validator script failed, so I removed it completely. I will try again

Reply:
The following screenshot shows what got me confused "release device repository" action is passed but "validate" action is failed. Probably I misunderstand the actions, but I find it somewhat confusing that they don't both pass. When I remove the regionalParameter then I get a pass on "validate" but not on "release device repository" 
![grafik](https://user-images.githubusercontent.com/1138612/186108842-df661729-eeeb-4790-92cb-819014bb9a81.png)

Reply:
> The following screenshot shows what got me confused "release device repository" action has passed but "validate" action has failed. Probably I misunderstand the actions, but I find it somewhat confusing that they don't both pass. When I remove the regionalParameter then I get a pass on "validate" but not on "release device repository" ![grafik](https://user-images.githubusercontent.com/1138612/186108842-df661729-eeeb-4790-92cb-819014bb9a81.png)

let me check